### PR TITLE
Refactor code.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

**Comment by terabytesoftw** [comment]
Of course the `composer.json` we will adjust it once merge https://github.com/yiisoft/db/pull/240

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
         "ext-mbstring": "*",
         "ext-pdo": "*",
         "yiisoft/arrays": "^1.0",
-        "yiisoft/db": "^3.0@dev",
+        "yiisoft/db": "dev-move-schemacache-to-schema-class as 3.0",
```
Comment (samdark) [inline composer.json:25]: Should be updated before merge.

File: tests/ConnectionTest.php [hunk 2]
```diff
@@ -43,26 +43,26 @@ public function testExceptionContainsRawQuery(): void
...
-        $db->setEnableProfiling(true);
+        $db->setLogger($this->logger);
+        $db->setProfiler($this->profiler);
 
         $this->runExceptionTest($db);
 
         /* profiling only */
-        $db->setEnableLogging(false);
-        $db->setEnableProfiling(true);
+        $db->setLogger();
```
Comment (samdark) [inline tests/ConnectionTest.php:52]: That looks weird. I'd require specifying `null` explicitly.

Comment (terabytesoftw) [reply tests/ConnectionTest.php]: done fixed.

**Review by samdark — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (terabytesoftw) [reply composer.json]: done.

**Pull request merged**